srcpkgs/python3-vint/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'python3-vint'
 pkgname=python3-vint
-version=0.3.19
-revision=6
-wrksrc="vim-vint-${version}"
+version=0.3.21
+revision=1
+wrksrc="vint-${version}"
 build_style=python3-module
 hostmakedepends="python3-setuptools"
 depends="python3-setuptools python3-ansicolor python3-chardet python3-yaml"
+checkdepends="python3-pytest ${depends}"
 short_desc="Lint tool for Vim script language (Python3)"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="MIT"
 homepage="https://github.com/Kuniwak/vint"
-distfiles="${PYPI_SITE}/v/vim-vint/vim-vint-${version}.tar.gz"
-checksum=aba8938f4c720e5c70a25f4dc7d845c5ce87aba0b778d1686f18b8da0c88f234
+distfiles="https://github.com/Vimjas/vint/archive/refs/tags/v${version}.tar.gz"
```

Comment:
Why does `$homepage` disagree with the base of `$distfiles`?

Reply:
Looks like https://github.com/Kuniwak/vint redirects to https://github.com/Vimjas/vint. I'll change it.